### ISSUE TITLE
There seems to be an issue with the satiety displayed on the server.

### ISSUE THREAD

**Issue by EmptyDreams** [squeek502/AppleSkin#304]
I installed AppleSkin on the client, but the module is not working properly.

The specific manifestation is that when I hold food, the module sometimes draws a overlay layer to indicate how much satiety I will recover, while other times it doesn't. Additionally, sometimes I have already switched items to non usable items, and AppleSkin still renders the overlay layer.

Currently, I have only encountered this issue when using the Fabric side, and AppleSkin is working properly when I am using Forge.

I know that if the server does not have AppleSkin installed, the module cannot guarantee obtaining correct data, but our server has not modified the relevant content, and even if the data is incorrect, AppleSkin should still render overlay layers.

My version is:
Minecraft: 1.20.1
Fabric API: 0.92.2
AppleSkin: 2.5.1

https://github.com/squeek502/AppleSkin/assets/44894906/7b038bbb-68f8-4c7f-8826-28848ebf693a

**Comment by squeek502** [comment]
Does the food you're holding show food values in its tooltip?

**Comment by EmptyDreams** [comment]
> Does the food you're holding show food values in its tooltip?您拿着的食物是否在其工具提示中显示食物价值？

Yes, when I use the mouse to hover over food, relevant information will be displayed in the tooltip.

I found that it seems that the overlay layer is only displayed when I have not fully regained my satiety after consuming food, and it does not disappear when I switch to other items.
